lib/asset-manifest-generator.js

```diff
@@ -98,7 +102,7 @@ AssetManifestGenerator.prototype.build = function() {
 
       if (supportedTypes.indexOf(assetType) !== -1) {
         manifest.bundles[bundleName].assets.push({
-          uri: path.join(prepend, entry),
+          uri: generateURI(prepend, entry),
```

Comment:
I actually made sure the function was called without the context to avoid relying on any state. If we find a need to do so later, we can add it in.